### PR TITLE
Add WG Device Management

### PR DESCRIPTION
**Pull request by johnbelamaric**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by aojea** [comment]
+1 from SIG Network

File: sigs.yaml
Comment (jackfrancis) [inline sigs.yaml:3343]: Is there a case here for adding Multicluster as a stakeholder SIG? I'm thinking in particular of use cases that rely upon hw acceleration at massive scale (spanning many clusters). Having a common standard to aid w/ scheduling and hw capability discovery across many clusters would greatly simplify this problem space.

cc @JeremyOT @pmorie @lauralorenz

Comment (dims) [reply sigs.yaml:3343]: We can always amend this document if it becomes necessary. Pretty sure we'll end up with more than the sigs listed here as participants.

File: wg-device-management/charter.md
Comment (dims) [inline wg-device-management/charter.md:3]: :)

**Comment by mrbobbytables** [comment]
/committee steering

Just ping steering when you're ready for it to be reviewed 👍

File: wg-device-management/charter.md
Comment (johnbelamaric) [reply wg-device-management/charter.md:3]: Yes, I wasn't clear from https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md whether the charter can follow merge of this or we should put it together now. That is, whether writing the charter should be the first thing we do as a WG, or we need it first.

Comment (mrbobbytables) [reply wg-device-management/charter.md:3]: charter should be the first step with scope and exit criteria

**Comment by johnbelamaric** [comment]
Ok, @kubernetes/steering-committee please take a look. Thank you!

**Comment by dims** [comment]
/lgtm

(wearning my sig-arch hat)

File: wg-device-management/charter.md
Comment (alculquicondor) [inline wg-device-management/charter.md:7]: We have notes about device support in WG Batch... I would suggest removing it from the charter as part of this PR. But worth bringing up the topic in a WG Batch meeting too.

Comment (johnbelamaric) [reply wg-device-management/charter.md:7]: Seems like that could be done in a follow-up PR. I don't want to gate this on that. We have some urgency to move forward.

Comment (BenTheElder) [reply wg-device-management/charter.md:7]: I think we can iterate on that part but it sounds like that's probably a quick-follow-up.

File: wg-device-management/charter.md
Comment (BenTheElder) [reply wg-device-management/charter.md:91]: nit
```suggestion
This working group adheres to the Roles and Organization Management outlined in [wg-governance]
```

**Review by BenTheElder — COMMENTED** [review]
generally LGTM, with nit(s)

EDIT: in my steering committee role :-)

File: wg-device-management/charter.md
Comment (BenTheElder) [reply wg-device-management/charter.md:91]: I think this is doubly confusing and we should fix this, we already need to re-apply lgtm and get approvals anyhow

Comment (johnbelamaric) [reply wg-device-management/charter.md:91]: oh, I had accepted Kevin's suggestion but I think it lost it when I rebased (since I didn't pull down the PR branch).

Comment (BenTheElder) [reply wg-device-management/charter.md:91]: with his suggestion it still says "sig" where it should say "working group" 😅

Comment (johnbelamaric) [reply wg-device-management/charter.md]: fixed

**Comment by BenTheElder** [comment]
+1
[steering]

**Comment by BenTheElder** [comment]
/approve
/hold
[holding for the rest of steering to comment and approve]

**Comment by pohly** [comment]
Thanks @BenTheElder for checking!

+1
[steering] (but beware that I am also part of the proposed WG, so a bit biased :shrug:)

File: wg-device-management/charter.md
Comment (soltysh) [reply wg-device-management/charter.md:7]: There will be overlaps between this WG, batch and serving naturally. I think it will boil down to proper coordination when those overlapping areas are being worked on, to accommodate the input from the other interested parties.

**Review by soltysh — APPROVED** [review]
lgtm with my steering hat on :)

**Comment by pacoxu** [comment]
just go through the kick off meeting agenda and find the scope part is not updated according to the agenda. 

/lgtm
overall

and btw is DRA ga one of exit criteria？ No mention dra in this readme.

**Comment by johnbelamaric** [comment]
> just go through the kick off meeting agenda and find the scope part is not updated according to the agenda.

The scope in the agenda doc is a click down more in detail than the scope listed here in the charter. I think all things in the agenda doc can fold under the things listed here.

> and btw is DRA ga one of exit criteria？ No mention dra in this readme.

Yes, the two DRA KEPs are ones we have already, and also any new ones that we decide are needed from this effort all would need to get to some terminal state (not necessarily GA!).

**Comment by johnbelamaric** [comment]
Summarizing steering committee votes so far:

- @BenTheElder - +1 [link](https://github.com/kubernetes/community/pull/7805#issuecomment-2076056686)
- @justaugustus - pending
- @mrbobbytables - pending
- @pacoxu - +1 [link](https://github.com/kubernetes/community/pull/7805#issuecomment-2077061137)
- @palnabarun - pending
- @pohly - +1 [link](https://github.com/kubernetes/community/pull/7805#issuecomment-2076578353)
- @soltysh - +1 [link](https://github.com/kubernetes/community/pull/7805#pullrequestreview-2022271610)

File: wg-device-management/charter.md
Comment (mrbobbytables) [reply wg-device-management/charter.md:7]: Just to update here, serving has updated their charter that  clarified their scope to not overlap with batch.

**Review by mrbobbytables — COMMENTED** [review]
/lgtm
+1 with steering hat on :)

File: wg-device-management/charter.md
Comment (kannon92) [inline wg-device-management/charter.md:97]: This seems a bit vague. 

I could see that wg-device-management should go away once DRA has reached GA (and maybe 2-3 vendors) are widely supported with DRA in GA.

To be clear, this does not mean work is done in this area but I think the urgency of this group will go down once DRA is out and supported in many places.

Comment (johnbelamaric) [reply wg-device-management/charter.md:97]: Maybe? I think saying "DRA reaches GA" is also vague. The whole of the functionality will be delivered incrementally, most likely. So there isn't one or two existing KEPs that will go to GA and we're done. I expect a few more things to come out of this.

Comment (kannon92) [reply wg-device-management/charter.md:97]: Yea, I'm not sure of a clear exit criteria then. In serving it was asked to come up with clear goals so we know when to call the working group complete.

Comment (kannon92) [reply wg-device-management/charter.md:97]: ref: https://github.com/kubernetes/community/pull/7823/files#r1581095409

Comment (johnbelamaric) [reply wg-device-management/charter.md]: I added a note saying we would consider disbanding when core DRA reaches GA.

**Review by pacoxu — APPROVED** [review]

**Review by palnabarun — APPROVED** [review]
Excited to see this happen. ✨ 

/lgtm
(with steering hat)

**Comment by johnbelamaric** [comment]
Woohoo! Thanks everyone.

/hold cancel

**Pull request merged**
